gutil/status_test.cc

```diff
@@ -30,8 +30,8 @@ namespace gutil {
 
 absl::Status ReturnStatusWithAnnotatedStream(std::string status_message,
                                              std::string streamed_message,
-                                             bool is_google3) {
```

Comment:
If this is not used anymore, can we just remove it ? Same for the next one too.

Reply:
> If this is not used anymore, can we just remove it ? Same for the next one too.

removed this change.

Reply:
I meant remove the unused arg.

Reply:
Can you pls reply to the comments and not resolve them

Reply:
> I meant remove the unused arg.

is_google3 arg is used/set this file. let me know if you need me to change /remove google keyword from this argument.

Reply:
Just rename this to is_google

Reply:
is_google3 is Replaced with is_google.